docs/developer_guide/getting_started.md

```diff
@@ -2,19 +2,15 @@
 
 ## What is monero-javascript?
 
-Monero-javascript is a Node.js-based Monero library for javascript.
+monero-javascript is a javascript library for producing Monero applications. The library is built on [an API model](https://moneroecosystem.org/monero-java/monero-spec.pdf) that aims to serve as an intuitive, consistent, and future-proof interface to the core Monero software. In addition to standard RPC wallet and daemon server queries, monero-javascript is capable of performing native wallet operations through WebAssembly (Wasm). The Wasm wallet enables developers to build trustless, client-side applications by eliminating the need to communicate with an rpc wallet server intermediary.
 
-The library derives its object and method hierarchy from [The Hidden Model](https://moneroecosystem.org/monero-java/monero-spec.pdf), a concise, uniform, and intuitive API specification that also serves as the foundation for the [monero-cpp](https://github.com/woodser/monero-cpp-library) and [monero-java](https://github.com/monero-ecosystem/monero-java) libraries.
-
-In addition to traditional RPC wallet communication, monero-javascript can manage wallets natively with WebAssembly (Wasm). Monero-javascript's Wasm wallet lets developers perform completely trustless, client-side wallet operations by eliminating the need for an intermediary wallet RPC server.
-
-![Monero-javascript hierarchy](../architecture.png?raw=true)*Monero-javascript manipulates XMR and communicates with the blockchain via three channels: RPC wallet servers, RPC daemon servers, and Wasm wallets.*  
+![monero-javascript hierarchy](../architecture.png?raw=true)*monero-javascript uses Monero through three channels: RPC wallet servers, RPC daemon servers, and Wasm wallets.*  
 
 ## Initial Setup
 
 ### Install Node.js and the Node package manager (npm)
 
-Make sure you have Node.js and npm installed; you need them to obtain and use the monero-javascript library. See the ["Node.js and npm"](https://github.com/monero-ecosystem/monero-javascript/blob/master/docs/developer_guide/installing_prerequisite_software.md#nodejs-and-npm) section of the prerequisite installation guide for step-by-step instructions for downloading and installing Node.js and npm.
+Make sure you have Node.js and npm installed; you need to install Node.js and npmbefore you can install and use the monero-javascript library. See the ["Node.js and npm"](https://github.com/monero-ecosystem/monero-javascript/blob/master/docs/developer_guide/installing_prerequisite_software.md#nodejs-and-npm) section of the prerequisite installation guide for step-by-step instructions for downloading and installing Node.js and npm.
```

Comment:
`Node.js and npmbefore` => `Node.js and npm before`

Reply:
What the... this was one of the things I fixed! Darnit...

Reply:
Ah, perhaps that's why it says "outdated".